qa/standalone/osd/osd-peering.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+source $CEPH_ROOT/qa/standalone/ceph-helpers.sh
+
+function run() {
+    local dir=$1
+    shift
+
+    export CEPH_MON="127.0.0.1:7155" # git grep '\<7155\>' : there must be only one
+    export CEPH_ARGS
+    CEPH_ARGS+="--fsid=$(uuidgen) --auth-supported=none "
+    CEPH_ARGS+="--mon-host=$CEPH_MON"
+
+     local funcs=${@:-$(set | sed -n -e 's/^\(TEST_[0-9a-z_]*\) .*/\1/p')}
+    for func in $funcs ; do
+        $func $dir || return 1
+    done
+}
+
+ function TEST_fast_info() {
+    local dir=$1
+
+    setup $dir || return 1
+    run_mon $dir a --osd_pool_default_size=3 || return 1
+    run_mgr $dir x || return 1
+    run_osd $dir 0 || return 1
+    run_osd $dir 1 || return 1
+
+    ceph osd primary-affinity 1 10	# let osd.1 be primary
```

Comment:
Seems this will cause non-deterministic test results.  @Aran85 
There is a get_primary() and get_not_primary() ceph-helpers.sh function which I think you can use to achieve the same goal and eliminate the non-determinism...

Reply:
the the primary-affinity is want try to make the osd.1 be primary at last round when osd.0 already up..because the assert is on code block that only primary run ..